code/theme-default.css

```diff
@@ -0,0 +1,127 @@
+   body {
+     background-color: black;
+   }
+
+   h1 {
+     text-align: center;
+     color: white;
+   }
+
+   /* box for entire calculator - needed to position the content */
+
+   #id-container {
```

Comment:
don't use id for styling ... just classes

Reply:
I fixed it classes for styling only, will remember this.  id-container goes as rcalc-container and id-calculator goes as rcalc-calculator